modules/aggregation-layers/src/utils/gpu-grid-aggregation/aggregate-to-grid-vs.glsl.js

```diff
@@ -47,8 +51,17 @@ void main(void) {
 
   vec2 pos = project_to_pixel(windowPos);
 
+  vec2 pixelXY64[2];
```

Comment:
Looks like you are not handling `positions64Low` for `ScreenGridLayer`?

Reply:
`pixelXY64` defined here only to perform fp64 arithmetic for division operation, applies to both world space aggregation and screen space, but I am not actually using low fp64 positions attributes, doesn't seem to be needed, we can add if there is a need.

Reply:
The projection to clipspace is inaccurate if you don't use the low parts. This applies to both geospatial and non-geospatial.

Reply:
I'm only making the comment here because GitHub doesn't allow me to comment on a line you didn't change.

Reply:
Updated attributes to be 64bit and added attribute to the shader.